app/src/main/java/edu/calvin/cs262/teamc/MatchInfo.java

```diff
@@ -1,5 +1,11 @@
 package edu.calvin.cs262.teamc;
 
+/** MatchInfo
+ *
+ * Contains information on matches
+ *
+ * @author Andrw Thomas
```

Comment:
You spelled your own name wrong, lol. I'm guessing this was not intentional.

Reply:
bahahahahahaha